### PR TITLE
11.09.alpha.1 release prep

### PR DESCRIPTION
**Pull request by tswicegood**
_Not ready for merge_

### CODE REVIEW

**Comment by tswicegood** [comment]
I need to wrap up the generic foreign key stuff for in arm_wells prior to merging this.

File: README.rst [hunk 1]
```diff
@@ -46,35 +56,51 @@ structure for you.  Create a new project like this::
     $ armstrong init mysite
     armstrong initialized!
 
+You can initialize a project using the ``--demo`` parameter to initialize with
+a demo SQLite3 database already set up with.  This provides a working example
```
Comment (dmclain) [inline README.rst]: the second 'with' is a typo

File: README.rst [hunk 2]
```diff
@@ -46,35 +56,51 @@ structure for you.  Create a new project like this::
...
-The ``config.defaults`` module contains all of the base settings that are
-common to your environment.  ``config.development`` has settings specific to
-your development environment, while ``config.production`` contains all of your
-production settings.  ``config.urls`` is configured as the root URL
-configuration for your project.
+The ``settings`` directory contains your Django settings.  The
+``settings.defaults`` module contains all of the base settings that are common
+to your environment.  ``settings.development`` has settings specific to your
+development environment, while ``settings.production`` contains all of your
+production settings.
```
Comment (dmclain) [inline README.rst:93]: Should we mention that we also look for a development_local and production_local file that ought to be (git|svn|whatever)ignored

Comment (tswicegood) [inline README.rst:93]: Yeah, we should actually.  Want to take a stab at it?

File: setup.py [hunk 3]
```diff
@@ -19,13 +19,13 @@ setup(
...
+        "armstrong.core.arm_content>=0.4.1",
         "armstrong.core.arm_layout>=0.1.0",
         "armstrong.core.arm_sections>=0.2.1",
-        "armstrong.core.arm_wells>=0.2",
-        "armstrong.apps.articles>=0.1.0,<0.2",
-        "armstrong.apps.content>=0.1.0,<0.2",
+        "armstrong.core.arm_wells>=0.2.1",
+        "armstrong.apps.articles>=0.3.1",
+        "armstrong.apps.content>=0.1.0",
         "armstrong.apps.events>=0.1.2,<0.2",
```
Comment (dmclain) [inline setup.py:29]: all the other packages lost their <next minor release

**Pull request merged**